admin/translations/fr.json

```
@@ -27,17 +27,17 @@
         "triggers": {
           "tabName": "Déclencheurs",
           "prerequisite": {
-            "title": "Objectif à atteindre",
+            "title": "Pré-requis",
```

Je crois qu'il n'y a pas le tiret
```suggestion
            "title": "Prérequis",
```